google/cloud/forseti/services/dao.py

```diff
@@ -616,7 +616,8 @@ def expand_special_members(cls, session):
                         roles=[role])
                     LOGGER.info('iam_policy: %s', iam_policy)
                 except NoResultFound:
-                    LOGGER.debug('Found a project in non-existent IAM binding.')
+                    LOGGER.warning('Found a project in non-existent IAM binding'
```

Comment:
Nit:

'Found a non-existent project, or project outside of the organization, in an IAM binding: %s'

Reply:
Done